timApp/static/scripts/tim/velp/annotation.component.ts

```diff
@@ -516,11 +516,14 @@ export class AnnotationComponent
     }
 
     clearColor() {
-        this.values.color = "";
+        this.values.color =
+            this.original.color == null ? "" : this.original.color;
```

Comment:
Pieni refaktorointi:

```ts
this.values.color = this.original.color ?? "";
```

(tarkoittaa, että valitaan `this.original.color` jos se ei ole `null` eikä `undefined`, muussa tapauksessa valitaan  "oletusarvo" `""`).

Reply:
Joo, ei tullut vielä lihasmuistista tuota Typescriptin versiota :) Korjattu nyt.